### PR TITLE
Use composer-require-checker

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: .composer-require-checker.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "symbol-whitelist": [
+        "PhpCsFixer\\PhpunitConstraintXmlMatchesXsd\\Constraint\\XmlMatchesXsd",
```
Comment (keradus) [inline .composer-require-checker.json:3]: aren't we exposing this? why we need to whitelist it ?

File: composer.json [hunk 2]
```diff
@@ -14,6 +14,8 @@
   ],
   "require": {
     "php": "^5.5 || ^7.0",
+    "ext-dom": "*",
+    "ext-libxml": "*",
```
Comment (keradus) [inline composer.json]: are those really needed ?

Comment (kubawerlos) [reply composer.json]: composer-require-checker reported them missing

Comment (keradus) [reply composer.json]: ha, now i'm shocked, libxml, sure, but dom? :(

**Pull request merged**